src/resources/boot.lua

```diff
@@ -172,6 +172,12 @@ function lovr.boot()
   return lovr.run()
 end
 
+function lovr.permission(permission, granted)
+  if permission == "audio_capture" and granted then
+    lovr.audio.start("capture")
+  end
```

Comment:
Should this be implemented as default behavior?  Will other permissions have default behavior?  Hmm, it's interesting to think about.

Reply:
I think this is the least surprising implementation for a game engine; you ask for audio, user says yes, you get audio. I think it should either work this way, or have an explicit check-for-permission API so dev can check it up front...